docs/mps.rst

```diff
@@ -1,17 +1,12 @@
+.. currentmodule:: tensornetwork.matrixproductstates
+
 Matrix Product States
 ---------------------
 
-.. toctree::
-    :maxdepth: 2
-    :caption: Contents:
-	      
-.. autoclass:: tensornetwork.matrixproductstates.base_mps.BaseMPS
-    :members:
-    :noindex:
+.. autosummary::
+     :toctree: stubs
 
-.. autoclass:: tensornetwork.matrixproductstates.finite_mps.FiniteMPS
-    :members:
-       
-.. autoclass:: tensornetwork.matrixproductstates.infinite_mps.InfiniteMPS
-    :members:
+     tensornetwork.matrixproductstates.base_mps.BaseMPS
+     tensornetwork.matrixproductstates.finite_mps.FiniteMPS
+     tensornetwork.matrixproductstates.infinite_mps.InfiniteMPS
```

Comment:
Can you remove the `matrixproductstates.{base_mps, finite_mps,...}`? This _should_ just work with `tensornetwork.BaseMPS` etc.

Reply:
Done ^^